manual-deployment.md

```diff
@@ -211,17 +211,27 @@ For a list of recommended drivers that we've tested, see [Install Client Drivers
 
 ### 8. Monitor your cluster
 
-The CockroachDB Admin UI lets you monitor cluster-wide, node-level, and database-level metrics and events. To start up the Admin UI, point your browser to the URL in the `admin` field listed in the standard output of any node on startup, for example:
+The CockroachDB Admin UI lets you monitor cluster-wide, node-level, and database-level metrics and events. To view the secured Admin UI on remote node1.example.com, first establish an ssh tunnel, then point your browser to the URL in the `admin` field listed in the standard output of any node on startup. For example, your start command may have gaven output like this:
 
 ~~~ shell
-$ cockroach start --insecure --host=node1.example.com
+$ cockroach start --http-addr=127.0.0.1 --ca-cert=ca.cert --cert=node1.cert --key=node1.key --host=node1.example.com
 build:     {{site.data.strings.version}} @ {{site.data.strings.build_time}}
-admin:     https://node1.example.com:8080 <-------------------------------- USE THIS URL
+admin:     https://127.0.0.1:8080 <-------------- ESTABLISH SSH TUNNEL TO HERE
 sql:       postgresql://root@node1.example.com:26257?sslcert=%2FUsers%2F...
 logs:      cockroach-data/logs
 store[0]:  path=cockroach-data
 ~~~
 
+Here is how to use ssh to tunnel port 8081 from your desktop to the node1.example.com cluster node:
+
+~~~ shell
+$ ssh -L 8081:127.0.0.1:8080 node1.example.com
```

Comment:
throughout: remove all mention of ssh tunnels - we are not in the business of teaching people how to ssh. it's enough to say that the admin UI is only available on the interface specified by `--http-addr`.